runtime/caml/config.h

```diff
@@ -197,11 +197,7 @@ typedef uint64_t uintnat;
 #define Page_size (1 << Page_log)
 
 /* Initial size of stack (bytes). */
-#ifdef DEBUG
-#define Stack_size (32 * sizeof(value))
```

Comment:
I think this parameter is badly named which causes confusion: `Stack_size` -> `Stack_initial_size`

I have found it very useful for the initial stack size to be set deliberately smaller in `DEBUG` builds when touching any code involved with the dynamic stacks (e.g. stack caching, assembly stubs for stack reallocation, rewriting exception handler address on stack reallocation, etc.).

Given that the stack reallocation should be unobservable from an OCaml program (aside performance from the stack reallocation) is there any reason why it has to be the same between a `DEBUG` and non-`DEBUG` build?

Reply:
I agree with this, the small initial stack with the debug runtime also helped me detect problems with my changes.